### PR TITLE
update PYPI Descriptions

### PR DESCRIPTION
**Pull request by CrepeGoat**

### CODE REVIEW

**Review by StokesMIDE — CHANGES_REQUESTED** [review]
The title format needs to be changed, and the file type changed to `.md`

File: README.rst [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+enDAQ-calc is available on PYPI via `pip`::
+
+    pip install endaq-calc
+
+For the most recent features that are still under development, you can also use `pip` to install enDAQ-calc directly from GitHub::
+
+    pip install git+https://github.com/MideTechnology/endaq-python-calc.git@development
+
+
+Other Links
```
[the remark this answers is not in the document]
Comment (CrepeGoat) [reply README.rst]: ftr, I think no; we'll just add it in later when we have it

**Review by StokesMIDE — CHANGES_REQUESTED** [review]
The changes I'd like to see are to `setup.py`: change the description (to whatever is at the start of the README), version (1.0.0b1), status classifier (`Development Status :: 4 - Beta`), and URL (to endaq-python-calc). My line comments are just comments.

**Review by StokesMIDE — APPROVED** [review]
Approved. If any tweaks need to be done to pass the tests (which are still running atm), consider them proactively approved, too.

**Review by CFlaniganMide — APPROVED** [review]
looks like it's rendering correctly in github

**Pull request merged**